packages/dictionary/src/metaSchema/referenceSchemas.ts

```diff
@@ -23,7 +23,7 @@ export const ReferenceTag = zod
 	.string()
 	.regex(
 		RegExp('^#(/[-_A-Za-z0-9]+)+$'),
```

Comment:
should we scape forward slash?

Reply:
There are tests that show this is working. JS strings don't need to escape forward slash, and I don't think thats needed in regex either.

Reply:
Tests like this one: https://github.com/overture-stack/lectern/blob/develop/packages/dictionary/test/references.spec.ts#L53

Which references the following dictionary json with a reference tag: https://github.com/overture-stack/lectern/blob/develop/packages/dictionary/test/fixtures/references/simple_references/input.ts#L20